src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```
@@ -1593,7 +1593,7 @@ protected function runGitAction(
                         default => throw new \Exception('Invalid resource type')
                     };
 
-                    $comment = new Comment();
+                    $comment = new Comment($platform);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$platform'. (undefined)

(UnusedLocalVariable)